src/ExternalSearch.Providers.CVR/CVRExternalSearchProvider.cs

```diff
@@ -146,6 +140,16 @@ private IEnumerable<IExternalSearchQuery> InternalBuildQueries(ExecutionContext
                         yield return new ExternalSearchQuery(this, entityType, ExternalSearchQueryParameter.Identifier, result.ToString());
                 }
             }
+
+            if (config.ContainsKey(Constants.KeyName.OrgNameKey))
+            {
+                var organizationName = GetValue(request, config, Constants.KeyName.OrgNameKey, Core.Data.Vocabularies.Vocabularies.CluedInOrganization.OrganizationName);
+
+                foreach (var name in organizationName)
+                {
+                    yield return new ExternalSearchQuery(this, entityType, ExternalSearchQueryParameter.Name, name);
+                }
+            }
```

Comment:
This will blindly perform searches for any name. Even if the company resides in a different country.

> From Slack:
> Customer expected name search to work but it seems like this had been removed from the enricher some time ago.

Maybe go back in history and find why it was removed and look at the previous implementation.
Currently the `namePostFixFilter` is not used and would protect from allow search for name even without a country being specified for names that includes a danish name post fix.